tests/mesh/test-convert-solid.cpp

```diff
@@ -86,4 +88,73 @@ void test()
     splitter.split_solid_along_facets( {} );
 }
 
+void test_hybrid_solid()
+{
+    geode::OpenGeodeMeshLibrary::initialize();
+    std::vector< geode::Point3D > hex_points{
+        { { 0, 0, 0 } },
+        { { 1, 0, 0 } },
+        { { 0, 1, 0 } },
+        { { 1, 1, 0 } },
+        { { 0, 0, 1 } },
+        { { 1, 0, 1 } },
+        { { 0, 1, 1 } },
+        { { 1, 1, 1 } },
+    };
+    auto hex = geode::SolidMesh3D::create();
+    auto builder_hex = geode::SolidMeshBuilder3D::create( *hex );
+    for( const auto& point : hex_points )
+    {
+        builder_hex->create_point( point );
+    }
+    builder_hex->create_polyhedron( { 0, 1, 2, 3, 4, 5, 6, 7 },
```

Comment:
You can use // NOLINTBEGIN(*-magic-numbers) and // NOLINTEND(*-magic-numbers) to remove clang warnings on magic numbers in tests